installation/local-hosting-vps/patreon_logviewer.md

```diff
@@ -0,0 +1,106 @@
+---
+description: Hosting the patreon logviewer on the cloud or on your own computer.
+---
+
+# Patreon logviewer
+
+This logviewer hosting tutorial is written assuming you have already set up your bot and are running Ubuntu 20.04-22.04.
+
+## Setting up OAuth2
+
+To set up OAuth2, you need to set a few settings in your Discord Developer Portal. First, head to the [Discord Developer Portal](https://discord.com/developers/applications) and select your modmail bot application. Then go to "OAuth2" and "General"
+<figure><img src="https://i.imgur.com/ZqteRCn.png" alt=""></figure>
+
+First, copy the Client ID and Client Secret. You will need these later when filling in the .env.
```

Comment:
So far we have been referring to file names as `inline code`. Probably should change for consistency